config/v1/types/filesystem_test.go

```diff
@@ -77,118 +43,14 @@ func TestFilesystemFormatAssertValid(t *testing.T) {
 		},
 		{
 			in:  in{format: FilesystemFormat("")},
-			out: out{err: ErrFilesystemInvalidFormat},
-		},
-	}
-
-	for i, test := range tests {
-		err := test.in.format.AssertValid()
-		if !reflect.DeepEqual(test.out.err, err) {
-			t.Errorf("#%d: bad error: want %v, got %v", i, test.out.err, err)
-		}
-	}
-}
-
-func TestMkfsOptionsUnmarshalJSON(t *testing.T) {
-	type in struct {
-		data string
-	}
-	type out struct {
-		options MkfsOptions
-		err     error
-	}
-
-	tests := []struct {
-		in  in
-		out out
-	}{
-		{
-			in:  in{data: `["--label=ROOT"]`},
-			out: out{options: MkfsOptions([]string{"--label=ROOT"})},
-		},
-	}
-
-	for i, test := range tests {
-		var options MkfsOptions
-		err := json.Unmarshal([]byte(test.in.data), &options)
-		if !reflect.DeepEqual(test.out.err, err) {
-			t.Errorf("#%d: bad error: want %v, got %v", i, test.out.err, err)
-		}
-		if !reflect.DeepEqual(test.out.options, options) {
-			t.Errorf("#%d: bad format: want %#v, got %#v", i, test.out.options, options)
-		}
-	}
-}
-
-func TestFilesystemUnmarshalJSON(t *testing.T) {
-	type in struct {
-		data string
-	}
-	type out struct {
-		filesystem Filesystem
-		err        error
-	}
-
-	tests := []struct {
-		in  in
-		out out
-	}{
-		{
-			in:  in{data: `{"device": "/foo", "format": "ext4"}`},
-			out: out{filesystem: Filesystem{Device: "/foo", Format: "ext4"}},
-		},
-		{
-			in:  in{data: `{"format": "ext4"}`},
-			out: out{filesystem: Filesystem{Format: "ext4"}, err: ErrPathRelative},
```

Comment:
This is being dropped since it's covered by the path valiation?

Reply:
Yup